app/views/tutors/show.html.erb

```diff
@@ -420,27 +420,27 @@
 
               <div class="col-md-6">
                 <div class="panel panel-default">
-                  <div class="panel-heading">Crimial Record</div>
+                  <div class="panel-heading">Criminal Record</div>
```

Comment:
Nice catch!

Reply:
I have updated the .gitignore file to remove GemFile